youtube_dl/extractor/rai.py

```diff
@@ -189,6 +189,10 @@ def _real_extract(self, url):
 
         program_info = media.get('program_info') or {}
         season = media.get('season')
+        
+        year = media.get('track_info').get('year')
```

Comment:
This isn't safe: if `media.get('track_info')` isn't the expected `dict`, it will crash.